.github/workflows/build.yml

```diff
@@ -23,12 +23,12 @@ jobs:
           cache: "maven"
 
       - name: Install Missing Dependencies
-        run: sudo apt-get install -y libv4l-0 libopencv-dev python3-opencv
+        run: sudo apt-get install -y libv4l-0 libopencv-dev python3-opencv libgtk2.0-0
```

Comment:
what needs lib tk ? some native ui stuff?

Reply:
attempting to solve openvc issues on github actions - probably more of a resource issue vs dependency, but there might be a UI dependency